shelterAPI.json

```diff
@@ -11,19 +11,41 @@
           "Endpoints"
         ],
         "summary": "Gets all shelters.",
+        "parameters": [
+          {
+            "name": "pageNumber",
+            "in": "query",
+            "style": "form",
+            "schema": {
+              "type": "integer",
+              "format": "int32",
+              "description": "Number of requested page, indexed from 0. If null default value is assigned."
+            }
+          },
+          {
+            "name": "pageCount",
+            "in": "query",
+            "style": "form",
+            "schema": {
+              "type": "integer",
+              "format": "int32",
+              "description": "Number of items to return. If null default value is assigned."
+            }
+          }
+        ],
         "responses": {
           "200": {
             "description": "Success", 
             "content": {
               "application/json": {
                 "schema": {
-                  "type": "array",
-                  "items": {
-                    "$ref": "#/components/schemas/ShelterDTO"
-                  }
+                  "$ref": "#/components/schemas/GetSheltersResponse"
                 }
               }
             }
+          },
+          "400": {
+            "description": "Bad request - lack of mandatory query parameters."
```

Comment:
Currently no query parameters are mandatory - I would change the response description to just general "bad request - invalid request field values" or sth

Reply:
Forgot to delete that when changing parameters from being mandatory. Will fix